core/lib/libtomcrypt/src/ciphers/aes/aes_armv8a_ce.c

```diff
@@ -299,11 +299,17 @@ static int aes_ctr_encrypt_nblocks(const unsigned char *pt, unsigned char *ct,
 		return CRYPT_ERROR;
 	}
 
+	/* In big endian mode IV_size=0 -> 128 bits and IV_size=8 -> 64 bits */
```

Comment:
Still not clear to me. "IV_size=0 -> 128 bits": what is 128 bits? The counter?

Reply:
See my response below.